heron/statefulstorages/src/java/org/apache/heron/statefulstorage/localfs/LocalFileSystemStorage.java

```diff
@@ -32,20 +34,25 @@
 import org.apache.heron.spi.statefulstorage.IStatefulStorage;
 import org.apache.heron.spi.statefulstorage.StatefulStorageException;
 
-public class LocalFileSystemStorage implements IStatefulStorage {
+public class  LocalFileSystemStorage implements IStatefulStorage {
   private static final Logger LOG = Logger.getLogger(LocalFileSystemStorage.class.getName());
 
   private static final String ROOT_PATH_KEY = "heron.statefulstorage.localfs.root.path";
+  private static final String MAX_CHECKPOINTS_KEY = "heron.statefulstorage.localfs.max.checkpoints";
 
   private String checkpointRootPath;
+  private int maxCheckpoints;
 
   @Override
   public void init(Map<String, Object> conf) throws StatefulStorageException {
     checkpointRootPath = (String) conf.get(ROOT_PATH_KEY);
+    maxCheckpoints = (int) conf.get(MAX_CHECKPOINTS_KEY);
```

Comment:
could be helpful to have a default value?

Reply:
Providing a default value in code might cause confusion. I prefer to have this value comes from conf file. It's cleaner and easier to reason.

Reply:
Added the default value